src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -392,18 +388,18 @@ def _cancel_jobs(self, req: JobRequest) -> dict:
     def _retry_jobs(self, req: JobRequest) -> dict:
         job_id_list = self._get_job_ids(req)
         retry_results = self._jm.retry_jobs(job_id_list)
-        self.send_comm_message(RETRY, retry_results)
         retry_ids = [
             result["retry"]["jobState"]["job_id"]
             for result in retry_results.values()
             if "retry" in result
         ]
-
         if len(retry_ids):
             self.send_comm_message(
                 NEW,
                 {JOB_ID_LIST: retry_ids},
             )
+        self.send_comm_message(RETRY, retry_results)
```

Comment:
Switch order of NEW and RETRY messages -- the new message comes first, followed by the RETRY message.